docs/pages/toolchain/slot/deployment-commands/create.md

```diff
@@ -0,0 +1,29 @@
+# create
+
+The **`create`** command initializes and configures new deployments for services like Katana, Torii, and Madara. It sets up specific configurations for each service, enabling rapid deployment and testing environments.
+
+### **Usage**
+
+```sh
+slot deployments create <Project Name> <Service> [options]
+```
+
+### **Examples**
+
+1. **Create a Katana Deployment:**
+    
+    ```sh
+    slot deployments create 'MyProject' katana --version "1.0" --block-time 10
```

Comment:
We can remove the '' if the project has not spaces in it.
Same for the version. We can also update for a more recent version that exists:
```
slot d create example_project katana --version 1.0.0-alpha.1
```